src/lib/consts.ts

```diff
@@ -0,0 +1 @@
+export const lastEditedAtFormat = 'HH:mm dd/MM/yyyy'
```

Comment:
I'm not convinced by needing a whole separate file from this. It leads to more cluttered imports since it cannot be directly imported from `$lib`. For now, I'm suggesting to export function in `lib/index` along the lines of:
```ts
export function formatTimeFromCMS(time: string){
    ...
}
```
This leads to extra-flexibility in the exact formatting of the time and more strictly encourages uniformity in formatting.

Reply:
Very well